component-catalog/src/Examples/Menu.elm

```diff
@@ -457,18 +564,243 @@ view ellieLinkConfig state =
+            , value = .name
+            , width = Css.pct 20
+            , cellStyles = always [ Css.padding2 (Css.px 14) (Css.px 7), Css.verticalAlign Css.middle, Css.fontWeight Css.bold ]
+            , sort = Nothing
+            }
+        , Table.custom
+            { header = text "Example"
+            , view = \{ name, menuType, entries } -> forcedOpenExample name menuType entries
+            , width = Css.pct 20
+            , cellStyles =
+                \{ name } ->
+                    let
+                        buttonId =
+                            forcedOpenExampleButtonId name
+                    in
+                    [ Css.padding2 (Css.px 14) (Css.px 7)
+                    , Css.verticalAlign Css.middle
+                    , Css.Global.descendants
+                        [ -- when menus are open, we add this overlay to
+                          -- ensure that clicks close the menu.
+                          -- so if the menu is open, nothing else on the page
+                          -- is interactive by mouse.
+                          -- for the purposes of these static examples,
+                          -- this is very unhelpful, so we hide the overlay.
+                          --
+                          -- if changing this code, please ensure the example in the CC is still interactive.
+                          Css.Global.class "Nri-Menu-Overlay"
+                            [ Css.display Css.none
+                            ]
+                        , -- Menus are absolutely positioned, but this is
+                          -- pretty inconvenient for displaying them in a table.
+                          -- This override is so that these Menu will be part of
+                          -- the normal flow of the page, so that their
+                          -- content is always visible.
+                          Css.Global.selector ("#" ++ buttonId ++ " + div")
+                            [ Css.position Css.relative ]
+                        ]
+                    ]
+            , sort = Nothing
+            }
+        , Table.custom
+            { header = text "Pattern Notes"
+            , view = \{ about } -> Text.smallBody [ Text.markdown about ]
+            , width = Css.pct 60
+            , cellStyles =
+                always
+                    [ Css.padding2 (Css.px 14) (Css.px 7)
+                    , Css.verticalAlign Css.top
+                    , Css.Global.descendants
+                        [ Css.Global.pre [ Css.whiteSpace Css.preWrap ]
+                        ]
+                    ]
+            , sort = Nothing
+            }
+        ]
+        [ { name = "List of entries"
+          , menuType = Menu.navMenuList
+          , entries =
+                [ Menu.entry "list-of-entries-clickable-text-entry" <|
+                    \attributes ->
+                        ClickableText.button "ClickableText"
+                            [ ClickableText.small
```

Comment:
I think you can remove all these `small`s (assuming medium is the default)

Reply:
Oh great! I was going to slack you about this, nice 💯

Reply:
Without a size passed through, it now renders like this:
<img width="650" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/f7d7b54b-6f05-4417-ab53-76e8f11a450d">

I think this is because of the changes made to inherit size?

Should ClickableText entries always be ClickableText.medium?

Reply:
Why is it inheriting size if it's not using `appearsInline`?

Reply:
Per Slack thread, sounds like we should change the smalls to mediums!

Reply:
✅  Done, and I also set the clickable text menu-trigger size to an explicit Medium. Screenshot & description updated

Reply:
Ahh much nicer, thank you!